x/bank/types/key.go

```diff
@@ -55,3 +55,17 @@ func AddressFromBalancesStore(key []byte) (sdk.AccAddress, error) {
 func CreateAccountBalancesPrefix(addr []byte) []byte {
 	return append(BalancesPrefix, address.MustLengthPrefix(addr)...)
 }
+<<<<<<< HEAD
+=======
+
+// CreateDenomAddressPrefix creates a prefix for a reverse index of denomination
+// to account balance for that denomination.
+func CreateDenomAddressPrefix(denom string) []byte {
+	// we add a "zero" byte at the end - null byte terminator, to allow prefix denom prefix
+	// scan. Setting it is not needed (key[last] = 0) - because this is the default.
+	key := make([]byte, len(DenomAddressPrefix)+len(denom)+1)
+	copy(key, DenomAddressPrefix)
+	copy(key[len(DenomAddressPrefix):], denom)
+	return key
+}
+>>>>>>> 16a953cc9 (fix: x/bank/044 migrateDenomMetadata (#10239))
```

Comment:
same here

Reply:
yes, I know, was writing about it in the Discord yesterday night. Thanks!